src/vs/workbench/api/electron-browser/mainThreadDebugService.ts

```diff
@@ -133,7 +133,7 @@ export class MainThreadDebugService implements MainThreadDebugServiceShape, IDeb
 						logMessage: l.logMessage
 					}
 				);
-				this.debugService.addBreakpoints(uri.revive(dto.uri), rawbps);
+				this.debugService.addBreakpoints(uri.revive(dto.uri), rawbps, 'mainThreadDebugService');
```

Comment:
This actually means that the breakpoint was added by an extension, so for readability I would potentially change the context

Reply:
Good point. Wasn't sure where this was used. Updated.